pyro/contrib/oed/eig.py

```diff
@@ -106,46 +217,61 @@ def naive_rainforth_eig(model, design, observation_labels, target_labels=None,
     :rtype: `torch.Tensor`
     """
 
-    if isinstance(observation_labels, str):
+    if isinstance(observation_labels, str):  # list of strings instead of strings
         observation_labels = [observation_labels]
     if isinstance(target_labels, str):
         target_labels = [target_labels]
 
-    # Take N samples of the model
-    expanded_design = lexpand(design, N)
-    trace = poutine.trace(model).get_trace(expanded_design)
-    trace.compute_log_prob()
+    s = 0.
+    if N_seq > 1:
```

Comment:
what's `N_seq`? this and `independent_priors` missing from docstring

Reply:
I'll document it... `N_seq > 1` actually doesn't work rn

Reply:
remove it?

Reply:
yes that's sensible, we'll keep it on another branch for now